sdk/src/main/java/io/radar/sdk/Radar.kt

```diff
@@ -575,6 +579,96 @@ object Radar {
         locationManager.startTracking(options)
     }
 
+    /**
+     * Mocks tracking the user's location from an origin to a destination.
+     *
+     * @param[origin] The origin.
+     * @param[destination] The destination.
+     * @param[mode] The travel mode.
+     * @param[points] The number of mock location updates.
+     * @param[interval] The interval in seconds between each mock location update. A number between 2 and 60.
+     * @param[callback] An optional callback.
+     */
+    @JvmStatic
+    fun mockTracking(
+        origin: Location,
+        destination: Location,
+        mode: RadarRouteMode,
+        points: Int,
+        interval: Int,
+        callback: RadarTrackCallback?
+    ) {
+        if (!initialized) {
+            return
+        }
+
+        apiClient.getMock(origin, destination, mode, points, object : RadarApiClient.RadarMockApiCallback {
```

Comment:
I wonder if it might be useful to expose the result of this call via the SDK as well - customers could potentially run these points through other APIs like geocoding / search as well